tests/resources/test_item.py

```diff
@@ -685,6 +685,32 @@ def test_item_search_get_query_extension(app_client, load_test_data):
     )
 
 
+def test_item_search_pagination(app_client, load_test_data):
+    """Test format of pagination links on a GET search"""
+    test_item = load_test_data("test_item.json")
+    for x in range(20):
+        test_item["id"] = f"test_item_{x}"
+        resp = app_client.post(
+            f"/collections/{test_item['collection']}/items", json=test_item
+        )
+        assert resp.status_code == 200
+
+    params = {"limit": 5}
+    resp = app_client.get("/search", params=params)
+    assert resp.status_code == 200
+
+    resp_json = resp.json()
+    links = resp_json["links"]
+    assert links[0]["rel"] == "next"
+    assert links[0]["href"].startswith("http://testserver/search?")
```

Comment:
We shouldn't rely on link ordering, we should find the link by rel. E.g.


```suggestion
    resp_json = resp.json()
    link = next(l for l in resp_json["links"] if l["rel"] == "next)
    assert link["href"].startswith("http://testserver/search?")
```